vm/adb/adb.go

```diff
@@ -338,6 +338,14 @@ func (inst *instance) repair() error {
 	// Switch to root for userdebug builds.
 	inst.adb("root")
 	inst.waitForSSH()
+
+	// ADB connects to a phone and starts syz-fuzzer while the phone is still booting.
+	// 60 seconds delay should allow the phone to finish a boot process and start
+	// all services and processes. Otherwise syzkaller may create a race condition
+	// which doesn't allow the phone to finalize initialization.
+	log.Logf(0, "waiting 60 sec for system init to finish")
```

Comment:
This does not look like a good fit for log level 0. It will print line for every boot, and the line is not particularly useful/actionable, just some internal details.